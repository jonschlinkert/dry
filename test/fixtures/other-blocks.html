
{% fake "nothing" %}
header: "other-blocks.html"
{% endfake %}

{% block "header" %}
header: "other-blocks.html"
{% endblock %}

{% bar %}
header: "other-blocks.html"
{% endbar %}
