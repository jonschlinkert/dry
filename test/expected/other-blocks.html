
{% fake "nothing" %}
header: "other-blocks.html"
{% endfake %}

header: "other-blocks.html"
{% bar %}
header: "other-blocks.html"
{% endbar %}
